test/index.ts

```diff
@@ -41,8 +46,17 @@ export function runTests(tests: Test, cwd: string = process.cwd()): void {
         });
     }
 
-    if (shouldRunBenchmarks) {
-        describe(`benchmarks`, () => {});
+    if (shouldRunBenchmarks && canisterNames !== undefined) {
+        const canisterNamesArray = Array.isArray(canisterNames)
+            ? canisterNames
+            : [canisterNames];
+
+        canisterNamesArray.forEach((canisterName) => {
```

Comment:
I'm not sure we want to do it like this. Also we aren't running the benchmarks, we just collecting and outputting the nice formats of the benchmarks. I think we might want to rename some of this.

Reply:
done